### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by cgcardona**
Working my way through the docs and making changes to the docs as needed.

## TLDR

* Use `myKeychain` var instead of calling `keyChain()`.
* Pass in proper var to `hasKey` and `getKey`.
* Create `Buffer` of message.
* Fix `verify` call by removing passed in `signerPubk`.
* Remove extra characters at end of lines

### Use `myKeychain` var instead of calling `keyChain()`

Since we're already assigning the result of `avm.keyChain()` to the `myKeychain` above it seems cleaner to just use that var throughout the examples. This is just personal preference. 

### Pass in proper var to `hasKey` and `getKey`

The existing example is passing a nonexistent `myaddress` var to `hasKey` and `getKey`

```js
let exists = myKeychain.hasKey(myaddress); //returns true if the address is managed
let keypair = myKeychain.getKey(myaddress); //returns the keypair class
```

This update passes in the `newAddress1` var which is created earlier in the README.

### Create `Buffer` of message

[`keypair.sign`](https://github.com/ava-labs/slopes/blob/master/src/apis/avm/keychain.ts#L156) accepts a `Buffer` as an argument. The current README is passing in a `string` which is incorrect. This update creates a Buffer out of the `message` var.

### Fix `verify` call by removing passed in `signerPubk`

[`keypair.verify`](https://github.com/ava-labs/slopes/blob/master/src/apis/avm/keychain.ts#L174) only accepts 2 arguments. This PR removes the 3rd `signerPubk` arg

### Remove extra characters at end of lines

I use [Markdown lint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) and it's complaining about extra characters at the end of lines.

### CODE REVIEW

**Comment by collincusce** [comment]
Solid catches! docs.avalabs.org probably need the same updates.

**Comment by cgcardona** [comment]
I've [created a PR](https://github.com/ava-labs/docs/pull/3) to update docs.avalabs.org too.

**Comment by collincusce** [comment]
You're the man! Thanks!
